charts/demand-capacity-management/values.yaml

```diff
@@ -0,0 +1,191 @@
+    type: ClusterIP
+    port: 8080
+  autoscaling:
+    enabled: false
+  ingress:
+    enabled: false
+    className: ""
+    annotations: {}
+  resources:
+    limits:
+      cpu: 300m
+      memory: 1Gi
+    requests:
+      cpu: 200m
+      memory: 1Gi
+  nodeSelector: {}
+  tolerations: []
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: DoesNotExist
+            topologyKey: kubernetes.io/hostname
+  livenessProbe:
+    failureThreshold: 3
+    initialDelaySeconds: 300
+    periodSeconds: 30
+    successThreshold: 1
+    timeoutSeconds: 1
+    path: /actuator/health
+    port: 8080
+  readinessProbe:
+    failureThreshold: 3
+    initialDelaySeconds: 300
+    periodSeconds: 30
+    successThreshold: 1
+    timeoutSeconds: 1
+    path: /actuator/health
+    port: 8080
+  applicationSecret:
+    enabled: true
+    clientSecret: ""
+    clientId: ""
+  configmap:
+    create: true
+    data:
+      security_enabled: 'false'
+  elastic:
+    enabled: false
+    security:
+      tls:
+        restEncryption: false
+
+
+# Frontend Settings
+frontend:
+  name: "dcm-frontend"
+  image:
+    registry: "ghcr.io"
+    pullPolicy: Always
+    repository: ghcr.io/catenax-ng/tx-demand-capacity-mgmt/demand-capacity-mgmt-frontend
+    tag: ""
+  certificate:
+    host: "localhost"
+  appName: demand-capacity-management-frontend
+  imagePullSecrets: []
+  podAnnotations: {}
+  podSecurityContext:
+    fsGroup: 2000
+  securityContext:
+    allowPrivilegeEscalation: false
+    runAsNonRoot: true
+    runAsUser: 101
+    runAsGroup: 3000
+    capabilities:
+      drop:
+        - ALL
+  service:
+    type: ClusterIP
+    port: 8080
+  autoscaling:
+    enabled: false
+  ingress:
+    enabled: false
+    className: ""
+    annotations: {}
+  resources:
+    limits:
+      cpu: 500m
+      memory: 2Gi
+    requests:
+      cpu: 400m
+      memory: 2Gi
+  nodeSelector: {}
+  tolerations: []
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: DoesNotExist
+            topologyKey: kubernetes.io/hostname
+  livenessProbe:
+    failureThreshold: 3
+    initialDelaySeconds: 10
+    periodSeconds: 1000
+    successThreshold: 1
+    timeoutSeconds: 1000
+  readinessProbe:
+    failureThreshold: 3
+    initialDelaySeconds: 10
+    periodSeconds: 1000
+    successThreshold: 1
+    timeoutSeconds: 1000
+  applicationSecret:
+    enabled: false
+  configmap:
+    create: true
+  elastic:
+    enabled: false
+    security:
+      tls:
+        restEncryption: false
+
+postgres:
+  enabled: true
+  auth:
+    database: dcm
```

Comment:
The default config should hard code the postgres credentials used. 
Instead rely on the auto-generated credentials provided by the bitnami chart.
There have been good examples in a thread on our [mailing list](https://www.eclipse.org/lists/tractusx-dev/msg00184.html)